mne/bem.py

```diff
@@ -281,7 +281,7 @@ def make_bem_solution(surfs, verbose=None):
     Parameters
     ----------
     surfs : list of dict
-        The BEM surfaces to use.
+        The BEM surfaces to use (output from make_bem_model)
```

Comment:
(single backticks so it hopefully links to the function in doc, not 100% sure it will -- if not then double-backticks would be good)